modules/client-configuration/pages/autoinst-cdrom.adoc

```diff
@@ -28,32 +28,41 @@ For information about creating a profile, see xref:client-configuration:autoinst
 
 The {cobbler} [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO.
 Specifying the distribution with [option]``--distro`` is mandatory when running [command]``buildiso`` command.
+[option]``--iso`` is the output location:
 
 ----
 cobbler buildiso --iso=/path/to/boot.iso --distro=<your-distro-label>
 ----
 
-[IMPORTANT]
-====
-You must use distro and profile labels as listed by {cobbler}, and not simply as shown in the UI.
-====
-
+You must use distribution and profile labels as listed by {cobbler}, and not simply as shown in the UI.
 To list the names of distributions and profiles stored by {cobbler}, run the commands:
 
 ----
-# cobbler distro list
-# cobbler profile list
+cobbler distro list
+cobbler profile list
 ----
 
 The boot ISO includes all profiles and systems by default.
-You can limit which profiles and systems are used with the [option]``--profiles`` and [option]``--systems`` options.
-For example:
+You can limit which profiles and systems are used with the [option]``--profiles`` and [option]``--systems`` options:
 
 ----
 cobbler buildiso --systems="system1 system2 system3" \
-  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label> --distro=<your-distro-label>
+  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label>" \
+  --distro=<your-distro-label>
 ----
 
+
+With [option]``--esp`` you can enable the build boot ISOs with Secure Boot.
+You explicitly specify the EFI System Partition (ESP).
+By default, {cobbler} searches for the ESP, and generates one if not found.
```

Comment:
Well, the default Cobbler search doesn't work for SUMA, which is why we add this feature :)

```suggestion
By default, {cobbler} generates the ESP partition, which disables Secure Boot.
```